Error on Nexus Repository Pro v3.14.0-04

Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**
Currently we have Nexus Repository Manager v3.13.0 and nexus-repository-r plugin with v1.0.1 running. 
Further to upgrading to Nexus Repository Manager 3.14.0-04 version, we had placed the plugin under the newly installed version and updated the relevant config files.

* **What feature or behavior is this required for?**
We have currently configured some R repositories on Nexus 3.13.0 and was expecting the same behavior further to upgrading to Nexus 3.14.0.

* **How could we solve this issue? (Not knowing is okay!)**
We are currently getting the below error message:

2018-10-23 15:38:45,746-0400 INFO  [jetty-main-1]  *SYSTEM org.sonatype.nexus.extender.NexusLifecycleManager - Start KERNEL
2018-10-23 15:38:45,748-0400 INFO  [jetty-main-1]  *SYSTEM org.sonatype.nexus.internal.log.LogbackLoggerOverrides - File: /binrepo/nexus/nexus-work3/nexus3/etc/logback/logback-overrides.xml
2018-10-23 15:38:45,750-0400 INFO  [jetty-main-1]  *SYSTEM org.sonatype.nexus.internal.log.LogbackLogManager - Configuring
2018-10-23 15:38:45,765-0400 INFO  [jetty-main-1]  *SYSTEM org.sonatype.nexus.extender.NexusContextListener - Installing: [nexus-pro-feature/3.14.0.04]
2018-10-23 15:38:47,655-0400 ERROR [jetty-main-1]  *SYSTEM org.sonatype.nexus.extender.NexusContextListener - Failed to initialize context
org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.apache.felix.resolver.ResolutionError.toException(ResolutionError.java:42)
	at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:389)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:375)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:347)
	at org.apache.karaf.features.internal.region.SubsystemResolver.resolve(SubsystemResolver.java:216)
	at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:263)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1176)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl$1.call(FeaturesServiceImpl.java:1074)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-10-23 15:38:47,657-0400 ERROR [jetty-main-1]  *SYSTEM org.sonatype.nexus.bootstrap.osgi.BootstrapListener - Failed to initialize
java.lang.RuntimeException: org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.sonatype.nexus.extender.NexusContextListener.contextInitialized(NexusContextListener.java:181)
	at org.sonatype.nexus.bootstrap.osgi.ListenerTracker.addingService(ListenerTracker.java:47)
	at org.sonatype.nexus.bootstrap.osgi.ListenerTracker.addingService(ListenerTracker.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:941)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:870)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:318)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:261)
	at org.sonatype.nexus.bootstrap.osgi.BootstrapListener.contextInitialized(BootstrapListener.java:121)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:953)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:558)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:918)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:370)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1497)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1459)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:848)
	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:287)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:545)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at com.codahale.metrics.jetty9.InstrumentedHandler.doStart(InstrumentedHandler.java:92)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.server.Server.start(Server.java:419)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at org.eclipse.jetty.server.Server.doStart(Server.java:386)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.sonatype.nexus.bootstrap.jetty.JettyServer$JettyMainThread.run(JettyServer.java:274)
Caused by: org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.apache.felix.resolver.ResolutionError.toException(ResolutionError.java:42)
	at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:389)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:375)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:347)
	at org.apache.karaf.features.internal.region.SubsystemResolver.resolve(SubsystemResolver.java:216)
	at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:263)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1176)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl$1.call(FeaturesServiceImpl.java:1074)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-10-23 15:38:47,657-0400 WARN  [jetty-main-1]  *SYSTEM org.eclipse.jetty.webapp.WebAppContext - Failed startup of context o.e.j.w.WebAppContext@6595d05f{Sonatype Nexus,/repo,file:///trvapps/nexus/nexus-3.14.0-04/public/,UNAVAILABLE}
java.lang.RuntimeException: org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.sonatype.nexus.extender.NexusContextListener.contextInitialized(NexusContextListener.java:181)
	at org.sonatype.nexus.bootstrap.osgi.ListenerTracker.addingService(ListenerTracker.java:47)
	at org.sonatype.nexus.bootstrap.osgi.ListenerTracker.addingService(ListenerTracker.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:941)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:870)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:318)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:261)
	at org.sonatype.nexus.bootstrap.osgi.BootstrapListener.contextInitialized(BootstrapListener.java:121)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:953)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:558)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:918)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:370)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1497)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1459)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:848)
	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:287)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:545)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at com.codahale.metrics.jetty9.InstrumentedHandler.doStart(InstrumentedHandler.java:92)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.server.Server.start(Server.java:419)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at org.eclipse.jetty.server.Server.doStart(Server.java:386)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.sonatype.nexus.bootstrap.jetty.JettyServer$JettyMainThread.run(JettyServer.java:274)
Caused by: org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.apache.felix.resolver.ResolutionError.toException(ResolutionError.java:42)
	at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:389)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:375)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:347)
	at org.apache.karaf.features.internal.region.SubsystemResolver.resolve(SubsystemResolver.java:216)
	at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:263)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1176)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl$1.call(FeaturesServiceImpl.java:1074)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-10-23 15:38:47,667-0400 INFO  [jetty-main-1]  *SYSTEM org.eclipse.jetty.server.AbstractConnector - Started ServerConnector@65a59315{HTTP/1.1,[http/1.1]}{10.48.5.255:8080}
2018-10-23 15:38:47,667-0400 ERROR [jetty-main-1]  *SYSTEM org.sonatype.nexus.bootstrap.jetty.JettyServer - Failed to start
java.lang.RuntimeException: org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.sonatype.nexus.extender.NexusContextListener.contextInitialized(NexusContextListener.java:181)
	at org.sonatype.nexus.bootstrap.osgi.ListenerTracker.addingService(ListenerTracker.java:47)
	at org.sonatype.nexus.bootstrap.osgi.ListenerTracker.addingService(ListenerTracker.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:941)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:870)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:318)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:261)
	at org.sonatype.nexus.bootstrap.osgi.BootstrapListener.contextInitialized(BootstrapListener.java:121)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:953)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:558)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:918)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:370)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1497)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1459)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:848)
	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:287)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:545)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at com.codahale.metrics.jetty9.InstrumentedHandler.doStart(InstrumentedHandler.java:92)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:138)
	at org.eclipse.jetty.server.Server.start(Server.java:419)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:108)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:113)
	at org.eclipse.jetty.server.Server.doStart(Server.java:386)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.sonatype.nexus.bootstrap.jetty.JettyServer$JettyMainThread.run(JettyServer.java:274)
Caused by: org.osgi.service.resolver.ResolutionException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=nexus-pro-feature; type=karaf.feature; version="[3.14.0.04,3.14.0.04]"; filter:="(&(osgi.identity=nexus-pro-feature)(type=karaf.feature)(version>=3.14.0.04)(version<=3.14.0.04))" [caused by: Unable to resolve nexus-pro-feature/3.14.0.04: missing requirement [nexus-pro-feature/3.14.0.04] osgi.identity; osgi.identity=nexus-repository-r; type=karaf.feature; version="[1.0.1,1.0.1]" [caused by: Unable to resolve nexus-repository-r/1.0.1: missing requirement [nexus-repository-r/1.0.1] osgi.identity; osgi.identity=org.sonatype.nexus.plugins.nexus-repository-r; type=osgi.bundle; version="[1.0.1,1.0.1]"; resolution:=mandatory [caused by: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-r/1.0.1: missing requirement [org.sonatype.nexus.plugins.nexus-repository-r/1.0.1] osgi.wiring.package; filter:="(osgi.wiring.package=se.sawano.java.text)"]]]
	at org.apache.felix.resolver.ResolutionError.toException(ResolutionError.java:42)
	at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:389)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:375)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:347)
	at org.apache.karaf.features.internal.region.SubsystemResolver.resolve(SubsystemResolver.java:216)
	at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:263)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1176)
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl$1.call(FeaturesServiceImpl.java:1074)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-10-23 15:38:47,702-0400 INFO  [FelixStartLevel]  *SYSTEM org.sonatype.nexus.extender.NexusContextListener - Uptime: 8 seconds and 146 milliseconds (nexus-pro-edition/3.14.0.04)
2018-10-23 15:38:47,704-0400 INFO  [FelixStartLevel]  *SYSTEM org.sonatype.nexus.extender.NexusLifecycleManager - Stop KERNEL

* **Anything else?**
List of changes done, further to upgrading to Nexus 3.14.0

- Copied nexus-repository-r-1.0.0.jar under <nexus_install_location>/system/org/sonatype/nexus/plugins/nexus-repository-r folder
  [The above jar file was copied from the previous Nexus Repository Instance]

	+-- nexus-repository-r
¦   +-- 1.0.0
¦       +-- nexus-repository-r-1.0.0.jar

- Updated <nexus_install_location>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.14.0-04/nexus-pro-feature-3.14.0-features.xml file with the following
		
			  <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
			+	  <!-- Nexus Team - Adding the R Plugin -->
			+     <feature version="1.0.1" prerequisite="false" dependency="false">nexus-repository-r</feature>
				  <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-yum</feature>
			  </feature>
			
			And

		<!-- Nexus Team - Adding the R Plugin -->	
		<feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.0.1">
		  <details>org.sonatype.nexus.plugins:nexus-repository-r</details>
		  <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.0.1</bundle>
		</feature>
	 </features>

--- a/README.md
+++ b/README.md
@@ -55,6 +55,23 @@ If everything checks out, the bundle for R should be available in the `target` f
 
 After installing the plugin, we have detailed instructions on [how to get started here!](docs/R_USER_DOCUMENTATION.md)
 
+## Compatibility with Nexus Repository Manager 3 Versions
+
+The table below outlines what version of Nexus Repository the plugin was built against
+
+| Plugin Version | Nexus Repository Version |
+|----------------|--------------------------|
+| v1.0.0         | <3.8.0-02                |
+| v1.0.1         | >=3.8.0-02               |
+| v1.0.2         | >=3.14.0-04              |
+
+If a new version of Nexus Repository is released and the plugin needs changes, a new release will be made, and this
+table will be updated to indicate which version of Nexus Repository it will function against. This is done on a time 
+available basis, as this is community supported. If you see a new version of Nexus Repository, go ahead and update the
+plugin and send us a PR after testing it out!
+
+All released versions can be found [here](https://github.com/sonatype-nexus-community/nexus-repository-r/releases).
+
 ## Installing the plugin
 
 There are a range of options for installing the R plugin. You'll need to build it first, and
@@ -64,8 +81,6 @@ then install the plugin with the options shown below:
 
 Installations done via the Karaf console will be wiped out with every restart of Nexus Repository. This is a
 good installation path if you are just testing or doing development on the plugin.
-
-Of note, version 1.0.1 is for Nexus Repository 3.8.0-02 and above, please use 1.0.0 if you are using an older version.
 
 * Enable Nexus console: edit `<nexus_dir>/bin/nexus.vmoptions` and change `karaf.startLocalConsole`  to `true`.
 
@@ -77,7 +92,7 @@ and enter the bundle command after it has started:
   # sudo su - nexus
   $ cd <nexus_dir>/bin
   $ ./nexus run
-  > bundle:install file:///tmp/nexus-repository-r-1.0.1.jar
+  > bundle:install file:///tmp/nexus-repository-r-1.0.2.jar
   ```
   This will return text similar to "Bundle ID: 282".  You can double check this ID using bundle:list 
   (for org.sonatype.nexus.plugins:nexus-repository-r; it should be the last one).  Then start it using this 
@@ -90,7 +105,7 @@ and enter the bundle command after it has started:
 
 For more permanent installs of the nexus-repository-r plugin, follow these instructions:
 
-* Copy the bundle (nexus-repository-r-1.0.1.jar) into <nexus_dir>/deploy
+* Copy the bundle (nexus-repository-r-1.0.2.jar) into <nexus_dir>/deploy
 
 This will cause the plugin to be loaded with each restart of Nexus Repository. As well, this folder is monitored
 by Nexus Repository and the plugin should load within 60 seconds of being copied there if Nexus Repository
@@ -100,20 +115,20 @@ is running. You will still need to start the bundle using the karaf commands men
 
 If you are trying to use the R plugin permanently, it likely makes more sense to do the following:
 
-* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-r/1.0.1/nexus-repository-r-1.0.1.jar`
+* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-r/1.0.2/nexus-repository-r-1.0.2.jar`
 * If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
 * If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
    ```
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
-   +     <feature version="1.0.1" prerequisite="false" dependency="false">nexus-repository-r</feature>
+   +     <feature version="1.0.2" prerequisite="false" dependency="false">nexus-repository-r</feature>
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-yum</feature>
      </feature>
    ```
    And
    ```
-   + <feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.0.1">
+   + <feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.0.2">
    +     <details>org.sonatype.nexus.plugins:nexus-repository-r</details>
-   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.0.1</bundle>
+   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.0.2</bundle>
    + </feature>
     </features>
    ```

--- a/pom.xml
+++ b/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.10.0-04</version>
+    <version>3.14.0-04</version>
   </parent>
   <groupId>org.sonatype.nexus.plugins</groupId>
   <artifactId>nexus-repository-r</artifactId>
   <name>${project.groupId}:${project.artifactId}</name>
-  <version>1.0.1</version>
+  <version>1.0.2</version>
   <inceptionYear>2017</inceptionYear>
   <packaging>bundle</packaging>
 
@@ -50,6 +50,7 @@
     <dependency>
       <groupId>se.sawano.java</groupId>
       <artifactId>alphanumeric-comparator</artifactId>
+      <version>1.4.1</version>
     </dependency>
 
     <dependency>

